app.js

```diff
@@ -97,21 +128,48 @@ function totalVote(){
     productButton1.removeEventListener('click', handleButton1);
     productButton2.removeEventListener('click', handleButton2);
     productButton3.removeEventListener('click', handleButton3);
+    Pictures.createVotes();
     showResults();
 
   }
 }
 
 function pickNewPictures() {
-  button1 = allPictures[Math.floor(Math.random() * allPictures.length)];
+  var randomPictures = Pictures.uniqueImages();
+
+  // var button1 = allPictures[0];
+  // var button2 = allPictures[1];
+  // var button3 = allPictures[2];
+
+
+  //button1 = allPictures[Math.floor(Math.random() * allPictures.length)];
+  button1 = allPictures[randomPictures[0]];
   mallImg1.src = button1.url;
 
-  button2 = allPictures[Math.floor(Math.random() * allPictures.length)];
+
+  //button2 = allPictures[Math.floor(Math.random() * allPictures.length)];
+  button2 = allPictures[randomPictures[1]];
   mallImg2.src = button2.url;
 
-  button3 = allPictures[Math.floor(Math.random() * allPictures.length)];
+  //button3 = allPictures[Math.floor(Math.random() * allPictures.length)];
+  button3 = allPictures[randomPictures[2]];
   mallImg3.src = button3.url;
 
+  allPictures[randomPictures[0]].timesDisplayed++;
+  allPictures[randomPictures[1]].timesDisplayed++;
+  allPictures[randomPictures[2]].timesDisplayed++;
+
+  Pictures.url = allPictures[randomPictures[0]].url;
+  Pictures.url = allPictures[randomPictures[0]].url;Pictures.url = allPictures[randomPictures[0]].url;
+
+  Pictures.url = allPictures[randomPictures[1]].url;
+  Pictures.url = allPictures[randomPictures[1]].url;
+  Pictures.url = allPictures[randomPictures[1]].url;
+
+  Pictures.url = allPictures[randomPictures[2]].url;
+  Pictures.url = allPictures[randomPictures[2]].url;
+  Pictures.url = allPictures[randomPictures[2]].url;
```

Comment:
I'm not sure why you duplicated all of these lines three times; once was enough!